docs/src/xhtml/components/datepicker/index.xhtml

```diff
@@ -78,19 +86,28 @@
 								<td><code>{void}</code></td>
 								<td>Close the DatePicker</td>
 							</tr>
+							<tr>
```

Comment:
apparently this is the old way of writing these API tables, we have a nicer template implementation, here is an example:

```html
<div data-ts="DoxApi">
	<script type="application/json">
		{
			"name": "ts.ui.UserImageModel",
			"properties": [
				{
					"name": "name",
					"type": "string",
					"desc": "The users full name."
				},
				{
					"name": "src",
					"type": "string",
					"desc": "Optional user photo or avatar."
				},
				{
					"name": "size",
					"type": "number",
					"desc": "The image width and height."
				},
				{
					"name": "value",
					"type": "string|number",
					"desc": "Used when sorting the column."
				}
			]
		}
	</script>
</div>
```


Could you convert this `<table>` to be in this format, and maybe have a separate API table for the cell, in case we decide to extend it, it doesn't get lost